app/jobs/rate_limit_checker_job.rb

```diff
@@ -1,5 +1,6 @@
-class RateLimitCheckerJob < ApplicationJob
-  queue_as :rate_limit_checker
+class RateLimitCheckerJob
+  include Sidekiq::Worker
+  sidekiq_options queue: :default, retry: false
```

Comment:
any reason why we're moving this to the default queue? 

I see this move from DJ to Sidekiq as a good chance to rationalize queues (now we have one per job which is probably too many)

Reply:
One per job is way too many for Sidekiq, we will have to have only a handful of queues since that is what Sidekiq handles best. Sorting out queues is something that takes a little trial and error but to begin we will start with a default(low priority for those jobs that are not time sensitive), medium priority(those that are a little time sensitive), high priority(those that are kicked off from the UI and users expect near realtime to be done), mailer(keeping emails separate is always a good move bc they drain so fast).

Like I said, picking a queue for a job will be a little bit of guesswork to begin and honestly we don't have that many jobs right now for it to probably matter much but once the queues get going we will track their sizes and then can determine if things need to be moved around or other queues need to be created.

Reply:
Agree on everything. 3 priorities plus mailers and long running jobs ☀️